_topics/en/op_codeseparator.md

```diff
@@ -0,0 +1,150 @@
+---
+title: OP_CODESEPARATOR
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Scripts and Addresses
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **OP_CODESEPARATOR** is an opcode that changes what data is used when
+  a signature commits to a script.  The opcode has been available since
+  the original version of Bitcoin Script, but its use and behavior have
+  changed over time (with further changes having been proposed).
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  In Bitcoin 0.1, each spend's scriptSig and outpoint's scriptPubKey
```

Comment:
I enjoyed this section on the history. A sort of "Bitcoin Opech, but backwards in time"